src/Ilist.js

```diff
@@ -0,0 +1,14 @@
+function IList() {}
+
+IList.prototype.clear = function() {console.log('IList init'); }
+IList.prototype.toString = function() {console.log('IList init'); }
+IList.prototype.getSize = function() {console.log('IList init'); }
+IList.prototype.toSort = function() {console.log('IList init'); }
+IList.prototype.push = function() {console.log('IList init'); }
+IList.prototype.pop = function() {console.log('IList init'); }
+IList.prototype.shift = function() {console.log('IList init'); }
+IList.prototype.unshift = function() {console.log('IList init'); }
+IList.prototype.toSplice = function() {console.log('IList init'); }
+IList.prototype.toSlice = function() {console.log('IList init'); }
```

Comment:
What are toSort, toSplice and toSlice?? I do not remember such methods in HW4.
Why doesn't any function take arguments? push, unshift, for example?

Reply:
getSort, getSort, getSplice, getSlice?